float32 x   # Control with respect to the x axis of the
            # DJI::OSDK::Control::HorizontalCoordinate.
float32 y   # Control with respect to the y axis of the
            # DJI::OSDK::Control::HorizontalCoordinate.
float32 z   # Control with respect to the z axis, up is positive.

float32 q_w
float32 q_x
float32 q_y
float32 q_z

# float32 yaw #Yaw position/velocity control w.r.t. the ground frame.

# float32 posThresholdInM  #(Meter)
# float32 yawThresholdInDeg  #(Degree)
